frameworks/kafka/tests/test_jmx.py

```diff
@@ -0,0 +1,204 @@
+import pytest
+import random
+import string
+
+import sdk_cmd
+import sdk_tasks
+import sdk_install
+import sdk_security
+import sdk_utils
+import subprocess
+from tests import config
+
+
+def install_jmx_configured_kafka(self_signed_trust_store: bool = True):
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+    sdk_install.uninstall(config.PACKAGE_NAME, foldered_name)
+    test_run = random_string()
+    create_keystore_cmd = ["keytool", "-genkey", "-alias", "self-signed-cert", "-keyalg",
+                           "rsa", "-dname", "CN=myhost.example.com,O=Example Company,C=US",
+                           "-keystore", "/tmp/{}-self-signed-keystore.ks".format(test_run), "-storepass", "deleteme",
+                           "-keypass", "deleteme", "-storetype", "jks"]
+
+    out = subprocess.check_output(create_keystore_cmd)
+    print(out)
```

Comment:
Print.. to ease with debugging for failures?

Reply:
removing it now